src/main/resources/lang/english.lang

```diff
@@ -1265,6 +1265,28 @@ heal reasons:
 	wither: withering, withered, wither potion, a wither effect, wither effect, wither
 	wither_spawn: wither spawning, wither spawn, wither summoning, a wither summoning, a wither spawn, a wither spawning
 
+# -- BossBar --
```

Comment:
Just noting that if the lang file changes get merged before this PR, this section will need to be moved to default.lang :)

Reply:
Alright, thanks for the info